README.md

```diff
@@ -135,7 +135,7 @@ CaBot v2 has been migrated to ROS2 (humble). It uses Docker containers to mainta
     # Name: Used for camera name space "<Name>" and camera TF link "<Name>_link"
     #
     # ex) "D:192.168.1.50:rs1 D:192.168.1.51:rs2"
-  CYCLONEDDS_URI=/home/developer/cyclonedds_spdp.xml   # need to use cyclone dds SPDP message option for multiple hosts
+  CYCLONEDDS_ALLOW_MULTICAST=spdp   # need to use cyclone dds SPDP message option for multiple hosts
```

Comment:
I think Jetson may use `CYCLONEDDS_URI` directory. `CYCLONEDDS_ALLOW_MULTICAST` is only used in `launch.sh` which is not used in Jetson.

Reply:
Is `CYCLONEDDS_ALLOW_MULTICAST` required as it is always `spdp`?

Reply:
Jetson uses `CYCLONEDDS_URI` directly.
I will remove `CYCLONEDDS_ALLOW_MULTICAST` by preparing two XML files (with NetworkInterface, without NetworkInterface).